modules/installation-and-upgrade/pages/server-intro.adoc

```diff
@@ -4,23 +4,43 @@
 {productname} uses an [literal]``X.Y.Z`` versioning schema.
 To determine which upgrade procedure you need, look at which part of the version number is changing.
 
+[NOTE]
+====
+The version number below are just examples.
+Do not understand them as most recent available options.
+{suse} uses these numbers for illustrative purposes only!
+====
+
+
 
 Major Version Upgrade (X Upgrade)::
-* Upgrading to the next major version.
-For example, upgrading from 3.2 to 4.0 or to 4.1.
+Such a major upgrade usually is an upgrade from X.Y to X+1.0 or to X+1.1, where Y is the latest minor version of the X series.
+For example:
+
+* From version 3.2 to 4.0 or to 4.1 (upgrading directly from 3.2 to 4.2 or later is s not supported).
+* From 4.3 to 5.0 or, probably, to 5.1 (upgrading directly from 4.3 to 5.2 or later probably is not supported).
 
 * See xref:installation-and-upgrade:server-x.adoc[].
 
 Minor Version Upgrade (Y Upgrade)::
-* Upgrading to the next minor version.
-This is often referred to as a service pack (SP) migration.
-For example, upgrading from 4.1 to 4.3 or from 4.2 to 4.3.
+Upgrading to the next minor version.
+This is often referred to as a product migration, service pack migration, or SP migration.
+Such a minor upgrade is an upgrade from X.Y to X.Y+1.
+For example:
+
+* From 4.2 to 4.3.
+* From 5.0 to 5.1.
+
+If you take a closer look, you always upgrade from and to the latest patch level of the minor version.
+For example, from 4.2.12 to 4.3.8 or from 5.0.i to 5.1.j (when available).
 
 * See xref:installation-and-upgrade:server-y.adoc[].
 
 Patch Level Upgrade (Z Upgrade)::
 * Upgrading within the same minor version.
-This is often referred to as a maintenance update.
-For example, upgrading from 4.3.0 to 4.3.1.
+This is often referred to as a maintenance update or MU.
+For example, upgrading from 4.3.7 to 4.3.8 or from 5.0.0 to 5.0.1 (when available).
```

Comment:
Don't mentinon 5.X, since it's not released yet and we will refactor the upgrade process.